app/models/ResultModel.py

```diff
@@ -58,7 +57,18 @@ def get_all_results(project_id):
     def get_one_result(id):
         data = ResultSchema().dump(ResultModel.query.get(id))
         return data
-    
+
+    @staticmethod
+    def get_paginated_results(project_id, page_no,row_size):
+        try:
+            data = ResultModel.query.filter_by(project=project_id).paginate(page=int(page_no), per_page=int(row_size))
+            data = ResultSchema().dump(data.items, many=True)
+            total_results = ResultModel.query.filter_by(project=project_id).count()
+            data.append({"total_results" : total_results})
+            return data
```

Comment:
I think directly returning data, total_results is suitable instead of appending total_results into data

```suggestion
            total_results = ResultModel.query.filter_by(project=project_id).count()
            return data, total_results
```